web/js/init.js

```diff
@@ -613,5 +613,12 @@ $(document).ready(function(){
                 $('.l-unit').find('.ch-toggle').prop('checked', false);
                 $('.l-unit.selected').find('.ch-toggle').prop('checked', true);
             });
+            // todo: maybe give the save button id?
+            $("[data-id=vstobjects][data-action=submit]").on("click", function () {
+              let loadingAnimationEle = document.createElement("img");
+              loadingAnimationEle.src = "https://raw.githubusercontent.com/phpmyadmin/phpmyadmin/7a6d4e68d3f6238f837d92c84c80081436ff02cc/themes/original/img/ajax_clock_small.gif";
```

Comment:
We should not use external sources.

Reply:
how about edd2b843716214850f761811dd2b929d62101f55